plotly/offline/offline.py

```diff
@@ -236,13 +280,32 @@ def iplot(figure_or_data, show_link=True, link_text='Export to plot.ly',
 
     display(HTML(plot_html))
 
+    if image:
+        if image not in __IMAGE_FORMATS:
+            raise ValueError('The image parameter takes only the '
+                             'following format types: `png`, `jpeg`, '
+                             '`webp`, `svg`')
+        # if the check passes then download script injection will commence.
+        # Write script to download image of the plot
```

Comment:
:dolls: This comment could be cleaner. Also, you might not really need the second line here, it's clear from the code.